From 42e53a0aadb76dfa7c11cd3b14eb4a41efba9bbe Mon Sep 17 00:00:00 2001
From: Matthias Maier <tamiko@43-1.org>
Date: Tue, 11 Jan 2022 07:20:31 -0600
Subject: [PATCH] also build virtfs-proxy-helper

The Gentoo ebuild splits the qemu build into a softmmu, user and tool
phase in order to be able to build and link some of the qemu emulators
statically. This unfortunately has the consequence that we never
configure with "have_virtfs" and "have_tools" at the same time.

As a workaround, simply build the virtfs userland unconditionally. After
all, it is a tiny executable
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index fbe856700..d6918b04c 100644
--- a/meson.build
+++ b/meson.build
@@ -1390,7 +1390,7 @@ have_virtfs = (targetos == 'linux' and
     libattr.found() and
     libcap_ng.found())
 
-have_virtfs_proxy_helper = have_virtfs and have_tools
+have_virtfs_proxy_helper = have_tools and libcap_ng.found()


 
 if get_option('virtfs').enabled()
   if not have_virtfs
-- 
2.34.1

